packages/modular-scripts/src/esbuild-scripts/start/index.ts

```diff
@@ -159,6 +159,10 @@ class DevServer {
     }
     const entryPoint = indexFiles[0];
 
+    const define = Object.assign({}, this.env.stringified, {
```

Comment:
you're doing this in both start and build, can this be part of the esbuild config util?